pipelinewise/cli/alert_handlers/errors.py

```diff
@@ -0,0 +1,27 @@
+"""
+PipelineWise CLI - Alert handler exceptions
+"""
+
+
+class NotImplementedAlertHandlerException(Exception):
+    """
+    Exception to raise when attempted to use a not implemented alert handler class
+    """
+    def __init__(self, *args, **kwargs):
+        Exception.__init__(self, *args, **kwargs)
```

Comment:
Please use `super().__init__` instead of `Exception.__init__`, there are the same however, the practice is to use `super` to call the parent class.

Reply:
fixed by https://github.com/transferwise/pipelinewise/pull/467/commits/f74caa958765671301a56f81cfc7643bd269d81a